documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx

```diff
@@ -333,57 +357,100 @@ Note that the `X` multiplier heavily lowers the `config_score` when nodes are ou
 >
-> **node_epoch_rewards = <span style={{display: 'inline-block'}}><EpochRewardBudget /></span> * 1 / 240 * (stake_saturation * performance)**
+> Where: <br/>
+> **[total_epoch_reward_budget](https://validator.nymtech.net/api/v1/epoch/reward_params) = <span style={{display: 'inline-block'}}><EpochRewardBudget /></span>** <br/>
+> **<abbr title="In Naive rewarding the node work fraction is same for all nodes in the active set">node_work_fraction</abbr> = 1 / rewarded_set_size** <br/>
+> **[rewarded_set_size](https://validator.nymtech.net/api/v1/epoch/reward_params) = 240** <br/>
+> **[stake_saturation_level](https://validator.nymtech.net/api/v1/epoch/reward_params) = <span style={{display: 'inline-block'}}><StakeSaturation /></span>** <br/>
+> **&alpha; = <abbr title="&alpha; is a constant (0.3) working as a premium for nodes with higher self bond">0.3</abbr>**
+> 
+> Therefore: <br/>
+> **node_epoch_rewards = <span style={{display: 'inline-block'}}><EpochRewardBudget /></span> \* [node_performance](#node-performance-calculation) \* [node_stake_saturation](#stake-saturation) \* ( <abbr title="In Naive rewarding the node work fraction is same for all nodes in the active set">( 1 / 240 )</abbr> + <abbr title="&alpha; is a constant (0.3) working as a premium for nodes with higher self bond">0.3</abbr> \* ( (  <abbr title="The actual number of tokens in the bond. Node bond size is capped at stake saturation level.">node_bond_size</abbr> / <span style={{display: 'inline-block'}}><StakeSaturation /></span> ) / 240 ) ) \* 1 / ( 1 + <abbr title="&alpha; is a constant (0.3) working as a premium for nodes with higher self bond">0.3</abbr> )**
 </Callout>
 
-Performance and stake saturation play an equally decisive role in the size of rewards earned after the epoch. The closer a node is to maximum value (`1`) of each of these parameters, the more rewards it will get.
+Performance and stake saturation (both are a float between `0` and `1` representing percentage) play an equally decisive role in the size of rewards earned after the epoch. The closer a node is to maximum value (`1`) of each of these parameters, the more rewards it will get.
+
+<br />
+<AccordionTemplate name="✏️ Explanation & Example: Functionality of &alpha; constant">
```

Comment:
Annoying style problem (this seems to affect others also): in light mode some of the text is unreadable 

![image](https://github.com/user-attachments/assets/0b738ec3-f10e-458a-a2dd-e9ca43e641ea)

Reply:
Works fine in dark mode (switched to light mode when I got here which is why I only noticed this far down) 

![image](https://github.com/user-attachments/assets/1e846fe1-ac5f-4eb4-a421-c139709add98)